pom.xml

```
@@ -514,6 +514,12 @@
             <version>5.10.0</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.github.spotbugs</groupId>
+            <artifactId>spotbugs</artifactId>
+            <version>4.5.2</version>
```

Looks important release: https://github.com/spotbugs/spotbugs/releases/tag/4.5.3
```suggestion
            <version>4.5.3</version>
```